localsettings.default.php

```diff
@@ -0,0 +1,7 @@
+<?php // Copy to localsettings.php to be included
+
+$localsettings['twitter_handle']              = 'YOUR_DB_NAME_HERE';
```

Comment:
When you bootstrap the localsettings.php for travis you'll want to change the default values for these settings so they are unique and match the setting key.